scout/server/blueprints/cases/controllers.py

```diff
@@ -1088,11 +1088,12 @@ def update_default_panels(store, current_user, institute_id, case_name, panel_id
     store.update_default_panels(institute_obj, case_obj, user_obj, link, panel_objs)
 
 
-def update_clinical_filter_hpo(store, current_user, institute_id, case_name, hpo_clinical_filter):
+def update_clinical_filter_hpo(store, current_user, institute_obj, case_obj, hpo_clinical_filter):
     """Update HPO clinical filter use for a case."""
-    institute_obj, case_obj = institute_and_case(store, institute_id, case_name)
```

Comment:
Skip one extra call to `institute_and_case` and pass the objects instead. It is vital that it gets called once for each request, but it should not be have to be many.